packages/cli/src/lib/implementation/global.options.ts

```diff
@@ -19,7 +19,12 @@ export function yargsGlobalOptionsDefinition(): Record<
     },
     config: {
       describe:
-        'Path the the config file, e.g. code-pushup.config.ts. By default it loads code-pushup.config.(ts|mjs|js).',
+        'Path to config file, e.g. code-pushup.config.ts. By default it loads code-pushup.config.(ts|mjs|js).',
+      type: 'string',
+    },
+    tsconfig: {
+      describe:
+        'Path to a TypeScript project, to be used when loading config file.',
```

Comment:
isn't it TypeScript config? or is it considered a project naming wise?

Reply:
Good point, [according to the docs](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html), the `tsconfig.json` file defines a TypeScript _project_, but itself is a _config_. 

I've changed the descriptions to use _config_ instead of _project_.